STM32/Libraries/SPI/Src/ADS1120.c

```diff
@@ -234,16 +234,16 @@ int ADS1120Init(ADS1120Device *dev)
     stmSetGpio(dev->cs, false);
     if (reset(dev) != HAL_OK)
     {
-        ret = -1;
+        ret = 1;
```

Comment:
Kan ikke lige se hvorfor du skifter til 1 og ikke -1, men hvis det er en fejl status, så negative tal det mest normale :-)

Reply:
Jeg skifter fordi jeg indsætter fejlkoden i et enkelt variable (linje 76 Temperature.c), der således indeholder alle fejlkoderne. Hvis jeg benytter negative tal som Gnistrup gjorde, overskriver en fejlende ADS, den næste i rækken, hvis der ikke er fejl på den. 

Det kan selvfølgelig sagtens skrives om til at benytte negative fejlkoder, og ændrede det egentlig primært for bekvemmelighedens skyld. Sig til, hvis du har en holdning til at den skal være negativ, så ændrer jeg den :)

Reply:
Det er ikke noget problem. Bare en general bemærkning :-)